### PR TITLE
make nix find the right flake

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: terraform/install/main.tf [hunk 1]
```diff
@@ -10,6 +10,6 @@ resource "null_resource" "nixos-remote" {
     environment = {
       SSH_PRIVATE_KEY = var.ssh_private_key
     }
-    command = "nix run --extra-experimental-features 'nix-command flakes' ${path.module}#nixos-anywhere -- ${local.nixos_anywhere_flags}"
+    command = "nix run --extra-experimental-features 'nix-command flakes' path:$(readlink -f ${path.module})/../..#nixos-anywhere -- ${local.nixos_anywhere_flags}"
```
Comment (zimbatm) [inline terraform/install/main.tf:13]:
```suggestion
    command = "nix run --extra-experimental-features 'nix-command flakes' path:${dirname(dirname(abspath(path.module)))})#nixos-anywhere -- ${local.nixos_anywhere_flags}"
```
`readlink -f` is not available on macOS's BSD readlink

Comment (Mic92) [reply terraform/install/main.tf]: Looks like readlink was needed after all.

Comment (Mic92) [reply terraform/install/main.tf]: `path:` seems to fix detection of the flake.

**Comment by Mic92** [comment]
bors merge

**Pull request merged**